api/source/service/CollectionService.js

```diff
@@ -1670,7 +1670,7 @@ exports.writeStigPropsByCollectionStig = async function ({collectionId, benchmar
     let version, release
     if (defaultRevisionStr) {
       if (defaultRevisionStr !== 'latest') {
-        const revisionParts = /V(\d+)R(\d+(\.\d+)?)/.exec(defaultRevisionStr)
+        const revisionParts = dbUtils.extractRevisionDetails(defaultRevisionStr)
         version = revisionParts[1]
```

Comment:
Since you are returning just version and release from your new function, we should be assigning results[0] and [1] here now instead of 1 and 2.